CHANGELOG.md

```diff
@@ -0,0 +1,21 @@
+# Change log
+
+## 2018-09-?? v0.1.0
+
+* (breaking) Remove template command
```

Comment:
Just FYI in case you want it to be consistent, the ./release script uses '[breaking]'